Update Splitbutton.html

helmut8080 opened the pull request:
Adapted for Bootstrap 4: https://getbootstrap.com/docs/4.3/components/dropdowns/#split-button

helmut8080 committed:
Update Collapsible.html

Adapted to "card" class of bootstrap 4

helmut8080 committed:
Update Collapsible.java

Adpated to boostrap 4 
- "accordion" class for container div
- data-target, aria-controls, aria-labelledby added

helmut8080 committed:
Update AlignmentBehavior.java

Commit:
Merge remote-tracking branch 'upstream/wicket-8.x-bootstrap-4.x' into wicket-8.x-bootstrap-4.x

update from upstream

helmut8080 committed:
Update SplitButton.html

Update according to bootstrap 4
- ul to div
- li to wicket:container
- some attributes added

helmut8080 committed:
Update AlignmentBehavior.java

helmut8080 committed:
Update SplitButton.html

change back

helmut8080 committed:
Update SplitButton.html

Adapted to Bootstrap 4

helmut8080 commented:
Only the commit from May 19, 2019 should be merged. Sorry still have difficulties with Github

martin-g commented on bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/SplitButton.html:
```diff
@@ -4,13 +4,16 @@
-                    <a wicket:id="button"></a>
-                </li>
-            </ul>
+            <button type="button" wicket:id="button"></button>
```
why did you remove `class="btn"` ?

martin-g commented on bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/SplitButton.html:
```diff
@@ -4,13 +4,16 @@
+            <button type="button" wicket:id="button"></button>
+            <button type="button" class="dropdown-toggle dropdown-toggle-split" wicket:id="caret" 
+                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                <span class="sr-only">Toggle Dropdown</span>
```
`Toggle Dropdown` should be i18n-ed.

martin-g commented:
> Only the commit from May 19, 2019 should be merged. Sorry still have difficulties with Githu

You need to pull the latest version of this branch and push the merge commit. This should remove the extra commits.

helmut8080 committed:
Update SplitButton.html

Sorry took some time. - Crazy week. Yes you are right. class="btn" needs to stay. Removed Toggle Dropdown.

martin-g commented:
Thank you, @helmut8080 !
